capa/features/file.py

```diff
@@ -33,3 +33,4 @@ class FunctionName(Feature):
     def __init__(self, name, description=None):
         # value is function name
         super(FunctionName, self).__init__(name, description=description)
+        self.name = "function-name"
```

Comment:
This seemed like the best place to set the name but open to suggestions. Setting the name here makes for a cleaner display later and makes integration with explorer much simpler.

Reply:
yeah, i agree. lets just add a comment making explicit that this overrides the behavior in `Feature`
```suggestion
        # override the name property set by `capa.features.Feature`
        # that would be `functionname` (note missing dash)
        self.name = "function-name"
```